clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -44,6 +45,9 @@ interface FossIdRestService {
     companion object {
         val JSON_MAPPER = JsonMapper()
             .setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
+            // FossID has a bug in get_results/scan.
+            // Sometimes the match_type is "ignored", sometimes it is "Ignored'
```

Comment:
Please consistently use either `"` or `'`, and add a missing dot at the end of the sentence.